platform/src/server/http/ssl_config.ts

```diff
@@ -16,11 +21,12 @@ const sslSchema = object(
     certificate: maybe(string()),
     key: maybe(string()),
     keyPassphrase: maybe(string()),
-    keystore: object({
-      path: maybe(string()),
-      password: maybe(string())
-    }),
-    certificateAuthorities: maybe(arrayOf(string())),
+    certificateAuthorities: maybe(
+      oneOf([
+        arrayOf(string()),
+        string(),
+      ])
+    ),
     supportedProtocols: maybe(
       arrayOf(oneOf([literal('TLSv1'), literal('TLSv1.1'), literal('TLSv1.2')]))
     ),
```

Comment:
Just wondering, what about the `redirectHttpFromPort` setting? For context:

>Kibana will bind to this port and redirect all http requests to https over the port configured as `server.port`.

Reply:
Hmm, good point! I missed that - will see how I can handle that in new platform.